lua/nvim-tree/renderer.lua

```diff
@@ -260,6 +260,9 @@ local function update_draw_data(tree, depth, markers)
       end
       if not has_children then folder_hl = "NvimTreeEmptyFolderName" end
       set_folder_hl(index, offset, #icon, #name+#git_icon, folder_hl)
+      if vim.g.nvim_tree_highlight_opened_files and node.open then
+        set_folder_hl(index, offset, 0, #icon, "NvimTreeOpenedFolderName")
+      end
```

Comment:
now that i think of this, maybe instead of using the variable to highlight the folder icon when opened, maybe just doing
```lua
if not has_children..... end
if node.open then folder_hl = "NvimTreeOpenedFolderName" end
```
would suffice, and highlighting NvimTreeOpenenedFolderName as directory by default

Reply:
what i mean is that you would only need to override the highlight yourself to get highlighting on folder, not based on the variable value.

Reply:
Ok, I think I understood, but that would imply highlighting everything (not only the icon) including folder name. Is that a problem? If not I can simplify it as you suggested

Reply:
hum i guess it would make more sense indeed even though i didn't think about this. We should keep consistency so i'd say yes.

Reply:
Agreed. Also, the Highlight color now makes it more discrete even highlighting the whole foldername. Check it out